Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -0,0 +1,1571 @@
+      description: Is the device virtual machine
+      type: Boolean
+    - contextPath: CarbonBlackEEDR.Device.last_name
+      description: Last name.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.scan_status
+      description: Scan status.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.last_internal_ip_address
+      description: Last internal IP address.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.last_policy_updated_time
+      description: Last policy updated time.
+      type: Date
+    - contextPath: CarbonBlackEEDR.Device.last_contact_time
+      description: Device last contact time.
+      type: Date
+    - contextPath: CarbonBlackEEDR.Device.quarantined
+      description: Is the device quarantined.
+      type: Boolean
+    - contextPath: CarbonBlackEEDR.Device.virtualization_provider
+      description: Virtualization Provider.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.organization_name
+      description: Organization Name.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.ad_group_id
+      description: Active directory group ID.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.policy_name
+      description: Policy name.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.policy_override
+      description: Policy override.
+      type: Boolean
+    - contextPath: CarbonBlackEEDR.Device.first_name
+      description: First name.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.current_sensor_policy_name
+      description: Current sensor policy name.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.id
+      description: Device ID.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.av_status
+      description: av status.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.av_pack_version
+      description: av pack version.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.email
+      description: User email.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.os_version
+      description: Device OS version.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.av_product_version
+      description: AV product version.
+      type: String
+    - contextPath: CarbonBlackEEDR.Device.last_location
+      description: Device last location.
+      type: String
+    - contextPath: Endpoint.ID
+      description: The unique ID within the tool retrieving the endpoint.
+      type: String
+    - contextPath: Endpoint.OS
+      description: Endpoint OS.
+      type: String
+    - contextPath: Endpoint.OSVersion
+      description: OS version.
+      type: String
+    - contextPath: Endpoint.MACAddress
+      description: The MAC address of the endpoint.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: Whether to group results. Can be "true" or "false". The default
```

Comment:
no need to mention in the description the default, demisto does it
just need to add defaultValue